### DISCUSSION TITLE
Pages not rendering correctly

### DISCUSSION THREAD

**Discussion by zachnb** [tailwindlabs/tailwindcss#11892] [Help] [answered]
Hello, i recently installed react and tailwind css in my laravel project using vs code. When i run the dev server and view the page, icons and images are way oversized and content is displaying without any styling. I have included an example screen shot trying to use the free-to-use reigstration form code from the tailwindcss components.

![screen](https://github.com/tailwindlabs/tailwindcss/assets/96999736/572f0d85-d06b-4b53-8594-e0a6ab1fa1e7)

Anyone have any ideas?

Thanks.

**Comment by wongjn** [comment, marked as answer]
It seems like you may not have integrated Tailwind correctly. Consider checking you have followed [the installation steps for your build tool](https://tailwindcss.com/docs/installation/framework-guides) correctly.
